.betterer.eslint.config.js

```diff
@@ -122,11 +122,11 @@ module.exports = [
         'error',
         {
           selector: 'Identifier[name=localStorage]',
-          message: 'Direct usage of localStorage is not allowed. Use `Store` from @grafana/data instead.',
+          message: 'Direct usage of localStorage is not allowed. import store from app/core/store instead.',
```

Comment:
you can't do this in grafana-data/grafana-runtime/grafana-prometheus/grafana-ui, only in core. you'll need both kinds of messages i think 😅

Reply:
Fun times. I fix.